nextflow_aws_batch_trigger.py

```diff
@@ -0,0 +1,314 @@
+import boto3
+import base64
+import argparse
+import datetime
+import time
+import logging
+
+now = datetime.datetime.now()
```

Comment:
You shouldn't need this since the logging will include date and time in output formatting

Reply:
As suggested, removed this line.